modules/ROOT/pages/plugins/premium/export.adoc

```diff
@@ -6,23 +6,24 @@
 :pluginname: Export
 :plugincode: export
 
-include::partial$misc/requires_5_5v.adoc[] +
+include::partial$misc/requires_5_5v.adoc[]
+
 include::partial$misc/premiumplugin.adoc[]
 
-The {pluginname} plugin adds the ability to export content from the editor to a user's local machine in various formats. For a list of available exporters and information on what they support, see the <<exporters,Exporters section>>.
+The {pluginname} plugin adds the ability to export content from the editor to a user's local machine in various formats. For a list of available exporters and information on what they support, see the xref:exporters[Exporters section].
 
 == Interactive example
 
 To export the editor content, either:
 
 * From the _File_ menu, click _Export_ and select _PDF_.
-* Click the Export toolbar button (image:{baseurl}/images/icons/export.svg[Export icon: A page with an arrow from the center of the page to the right of the page]) and select _PDF_.
+* Click the Export toolbar button (image:icons/export.svg[Export icon: A page with an arrow from the center of the page to the right of the page]) and select _PDF_.
 
 liveDemo::export[]
 
 == Basic setup
 
-To add the {pluginname} plugin to the editor, add `+{plugincode}+` to the `plugins` option in the editor configuration.
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
 
 For example:
 
```

Comment:
Can't leave a comment on the exact line, but the source code block below uses attributes so it needs `subs="attributes+"` added.

Reply:
Done